draft-ietf-dnsop-ns-revalidation.mkd

```diff
@@ -263,8 +263,10 @@ Otherwise, the addresses cannot be assumed to be complete or even authoritativel
 Note that there may be outstanding address validation queries for the names of the authoritative NS RRset (from referral address validation queries).
 In those cases no new validation queries need to be made.
 
+Strictly revalidating referral and authoritative NS RRset responses {#strict}
+-------------------------------------------------------------------
 Resolvers may choose to delay the response to a triggering query until it can be verified that the answer came from a name server listening on an authoritatively acquired address for an authoritatively acquired name.
-This would offer the most trustworthy responses with the least risk for forgery or eavesdropping.
+This would offer the most trustworthy responses with the least risk for forgery or eavesdropping, however without fallback to lower ranked NS RRsets and addresses, there is no failure mitigation and a failed NS RRset validation query will then lead to a hard failure to query the referred to child zone.
```

Comment:
```suggestion
This would offer the most trustworthy responses with the least risk for forgery or eavesdropping, however without fallback to lower ranked NS RRsets and addresses, there is no failure mitigation and a failed NS RRset validation query, due to a broken child NS RRset or to malfunctioning child zone's authoritative servers,  will then lead to a hard failure to query the referred to child zone.
```